### PR TITLE
Fix order.chargeStatus calculated incorrectly

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it fixes the issue reported here: https://github.com/saleor/saleor/issues/14326

Port of changes from: https://github.com/saleor/saleor/pull/14371

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases

### CODE REVIEW

**Review by zedzior — APPROVED** [review]

File: saleor/celeryconf.py [hunk 1]
```diff
@@ -33,5 +33,6 @@ def setup_celery_logging(loglevel=None, **kwargs):
     ],
     related_name="saleor3_17",
 )
+
```
Comment (fowczarek) [inline saleor/celeryconf.py:36]: Unnecessary change.

**Review by fowczarek — APPROVED** [review]

**Review by kadewu — APPROVED** [review]
